src/v0/index.ts

```diff
@@ -101,18 +101,19 @@ function encodeYesNoVote(
   _contest: YesNoContest,
   contestVote: YesNoVote
 ): string {
-  switch (contestVote) {
-    case 'no':
-      return '0'
-    case 'yes':
-      return '1'
-    default:
-      throw new Error(
-        `cannot encode yesno vote, expected "no" or "yes" but got ${JSON.stringify(
-          contestVote
-        )}`
-      )
+  if (
+    !(contestVote instanceof Array) ||
+    contestVote.length > 1 ||
+    (contestVote.length === 1 && !['yes', 'no'].includes(contestVote[0]))
+  ) {
+    throw new Error(
+      `cannot encode yesno vote, expected ['no'] or ['yes'] but got ${JSON.stringify(
+        contestVote
+      )}`
+    )
   }
+
+  return contestVote[0] === 'yes' ? '1' : '0'
```

Comment:
This still implicitly encodes `[]` and `['no']` equivalently. Is that intended?

Reply:
oof, good point. How should we be encoding `[]`?

Reply:
It means "no vote", right? So I'm guessing for v0 it's just an empty string and for v1 it's an absent vote. I'll noodle on it.